src/media/js/header_footer.js

```diff
@@ -13,10 +13,11 @@ define('header_footer',
             $('#mkt-nav--site-header, #site-header, main #site-nav').remove();
 
             $('<div id="mkt-nav--site-header" class="mkt-nav--wrapper"></div>')
-                .append(nunjucks.env.render('header.html'))
-                .append(nunjucks.env.render('mkt_nav.html', {
-                    categories: categories
+                .append(nunjucks.env.render('header.html', {
+                    // TODO: remove after waffle.
+                    categories: require('categories')
```

Comment:
Because we're gonna remove it after we strip the flag, it's only used for the old dropdown